package org.ruby.userauthservive.exceptions;

public class UserAlreadyExistException {
}
